### ISSUE TITLE
default theme outputs dump()

### ISSUE THREAD

**Issue by Puddingboy** [AnimalDesign/bolt-translate#20]
Due to this being included in the template:
```
{% elseif record.fieldtype(key) not in ['templateselect'] and attribute(record, key) is not empty %}

                    {# No special case defined for this type of field. We just output it, if it's
                       a simple scalar, and 'dump' it otherwise. #}
                    <p><strong>{{ key }}: </strong>
                        {% if attribute(record, key) is iterable %}
                            {{ dump(attribute(record, key)) }}
                        {% else %}
                            {{ attribute(record, key) }}
                        {% endif %}
                    </p>

                {%  endif %}
```
It's not a major issue, but it may be a decent idea to make sure this output is not triggered, so that it doesn't look like things are not working correctly.

**Comment by SahAssar** [comment]
Just adding some context from what I said on IRC:

Well, previously you couldn't access the original values in a record that had translated values, I changed that and put those in a special value that is called delocalizedvalues, but bolt's default theme loops through all the values on a record and when it reaches that one it does not know how to display it so it simply dumps them. Whether it's a theme or extension issue is a bit of a grey area, but I generally think that having the dump appear in the default theme is a small price to pay to be able to access those values. I'll look into another way of doing it though.

**Comment by SahAssar** [comment]
Fixed in the 3.0 version that will be tagged soon.
